proposals/csharp-10.0/improved-definite-assignment.md

```diff
@@ -205,7 +205,7 @@ if (b ? x != null && Set(out y) : x != null && Set(out y))
 bool Set(out int x) { x = 0; return true; }
 ```
 
-This is an admittedly niche scenario, that compiles without error in the native compiler, but was broken in Roslyn in order to match the specification at the time. See [internal issue](http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/529603).
```

Comment:
@RikkiGibson @gewarren where would be a good place to leave this note? I get that we don't want to publish a page to docs that links to an internal defunct bug tracker, but we still likely want to keep the link somewhere.

Reply:
What about the proposal issue #4465? Maybe we could link to this paragraph and as well as the internal issue?

Reply:
Possibly in here? https://github.com/dotnet/roslyn/blob/main/docs/compilers/CSharp/Compiler%20Breaking%20Changes%20-%20VS2015.md